From: Jan Beulich <jbeulich@suse.com>
Subject: SUPPORT.md: PV display frontend is unsupported in "backend allocation" mode

This wasn't meant to be supported, but wasn't stated this way.

This is XSA-363.

Reported-by: Jan Belich <jbeulich@suse.com>
Signed-off-by: Jan Beulich <jbeulich@suse.com>

--- a/SUPPORT.md
+++ b/SUPPORT.md
@@ -414,7 +414,8 @@ Guest-side driver capable of speaking th
 
 Guest-side driver capable of speaking the Xen PV display protocol
 
-    Status, Linux: Supported
+    Status, Linux: Supported (outside of "backend allocation" mode)
+    Status, Linux: Experimental (in "backend allocation" mode)
 
 ### PV Console (frontend)
 
